web/packages/teleport/src/Discover/Database/CreateDatabase/useCreateDatabase.ts

```diff
@@ -294,7 +294,8 @@ export function useCreateDatabase() {
     return (
       createdDb.uri !== db.uri ||
       createdDb.awsRds?.accountId !== db.awsRds?.accountId ||
-      createdDb.awsRds?.resourceId !== db.awsRds?.resourceId
+      createdDb.awsRds?.resourceId !== db.awsRds?.resourceId ||
+      createdDb.awsRds?.vpcId !== db.awsRds?.vpcId
     );
```

Comment:
How are these fields chosen? E.g. why not `subnets`? Could be worthy of a comment.

Reply:
I missed the subnets
Updated, thank you